index.html

```diff
@@ -86,22 +93,23 @@ <h1>I'm Naan. <br>Glad to see <br>you!</h1>
                         <div>2015</div>
                     </div>
                 </div>
-                <div class ="project_description">A daily selection of 
-                    privately personalized reads; no accounts or 
-                    sign-ups required.
+                <div class ="project_description">Experimental content creation feature 
+                    that allows users to add to an existing story over the course 
+                    of a day without spamming their friends.
                 </div>
                 <div class="tags">
                     <div>html</div>
```

Comment:
- [ ] i can see you tried to implement some required changes by the previous  but still there are some few stuffs you are still missing here consider double checking this requested change and try to fix it in your mobile version

Reply:
That was not a required change. It was optional. Everything I was told to fix on the mobile version was optional. I dont understand why every reviewer keeps adding new requirements. @divinecharlotte